app/models/offline_transaction_refund.rb

```diff
@@ -3,5 +3,86 @@
 # License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
 # Full license explanation at https://github.com/houdiniproject/houdini/blob/master/LICENSE
 class OfflineTransactionRefund < ApplicationRecord
+	include Model::SubtransactionPaymentable
 	belongs_to :payment
+
+	delegate :gross_amount, :net_amount, :fee_total, :nonprofit, to: :payment
```

Comment:
Is this the best way to refer to the nonprofit? I think so because it comes from payment, but maybe I'm wrong.

Reply:
That's a way but it's not the best way.

A couple things to consider:
* `payment` is legacy. While the payment nonprofit there needs to match the one in the transaction, we're eventually going to use the one in the transaction only.
* `nonprofit` should be available via `has_one :nonprofit, through: :subtransaction_payment` which is included as a part of the [`Model::SubtransactionPaymentable` concern](https://github.com/wwahammy/houdini/blob/feature/event-publisher/offline-transaction-refund/app/models/concerns/model/subtransaction_paymentable.rb). By doing so, we should be able to run calls like `OfflineTransactionRefund.where(nonprofit: Nonprofit.find(1))` which is pretty slick!

Was there a specific reason that led to needing to add the delegate for `nonprofit` here instead?

Reply:
I'm not sure know why, when I was writing the tests I wasn't able to access `nonprofit` without doing it, but I just tested it again and it worked ò.ó

Reply:
Update: https://github.com/wwahammy/houdini/pull/14/files#discussion_r650074781